docs/lti.md

```diff
@@ -2,9 +2,135 @@
 
 Marsha can act as an LTI provider and supports version 1.0 of the standard.
 
-For developers' reference, here are examples of LTI launch requests sent by the most common LMS. If your LMS is missing from this list, please feel free to submit a Pull Request to add it to the list and we will make sure Marsha supports it.
+In order to connect an LTI consumer site (such as Moodle or Open edX) a matching passport needs to be created in
+Marsha.
 
-## Launch request sent by Open edX
+## LTI Deep Linking
+
+LTI content can be selected in Moodle through Deep Linking.
+
+### Admin setup
+
+#### Create external tool preset
+Logged as admin, go to `Site administration`
+![moodle_lti_select_admin_1](images/moodle_lti_select_admin_1.png)
+
+Go to `Plugins` tab, and click on `Manage tools` under `External Tools`
+
+![moodle_lti_select_admin_2](images/moodle_lti_select_admin_2.png)
+
+Click on `configure a tool manually`
+
+![moodle_lti_select_admin_3](images/moodle_lti_select_admin_3.png)
+
+Fill the form with following data
+
+Label                 | Value
+--------------------- | --------------------------------------------------
+Tool name             | marsha staging
+Tool URL              | https://staging.marsha.apps.openfun.fr/
```

Comment:
you can replace by `marsha.education`. I think it's better for an example than our staging.